setup.py

```diff
@@ -20,6 +20,6 @@ def get_long_description() -> str:
     url="https://github.com/sile/kurobako-py",
     license="MIT",
     packages=find_packages(),
-    install_requires=["numpy"],
+    install_requires=["numpy", "packaging"],
```

Comment:
I added `packaging` in `install_requires` since Optuna also uses this library (See the discussion at https://github.com/optuna/optuna/issues/1560).

Reply:
On second thought, packaging should not be listed here because Optuna is not listed. I'll revert this change :bow: